source/_templates/about.html

```diff
@@ -1,11 +1,10 @@
-<h1><a href="https://nix.dev">nix.dev</a></h1>
+<div class="logo">
+  <a href="/">
+    <img src="/_static/img/nix.svg" alt="Nix snowflake logo">
```

Comment:
```suggestion
    <img src="/_static/img/nix.svg" alt="">
```

One can consider this a decorative element, which then should have an empty `alt` tag. Otherwise screen readers will say "Nix snowflake logo nix dot dev", which is not helpful at all.